### PR TITLE
Update console if no wallet file for full node and light wallets.

### PR DESCRIPTION
**Pull request by mikedennis**

### CODE REVIEW

File: Stratis.Bitcoin.Features.Wallet/IWalletManager.cs [hunk 1]
```diff
@@ -210,6 +210,12 @@ public interface IWalletManager : IDisposable
         /// <returns>The wallet filenames, along with the folder in which they're contained.</returns>
         (string folderPath, IEnumerable<string>) GetWalletsFiles();
 
+        /// <summary>
```
Comment (bokobza) [inline Stratis.Bitcoin.Features.Wallet/IWalletManager.cs]: I would rather have AreWallets be called ContainsWallets or something like that.

File: Stratis.Bitcoin.Features.Wallet/WalletManager.cs [hunk 2]
```diff
@@ -402,6 +402,15 @@ public int LastBlockHeight()
             return this.Wallets.Min(w => w.AccountsRoot.SingleOrDefault(a => a.CoinType == this.coinType)?.LastBlockSyncedHeight) ?? 0;
         }
 
+        /// <inheritdoc />
+        public bool AreWallets
```
Comment (bokobza) [inline Stratis.Bitcoin.Features.Wallet/WalletManager.cs]: Can we have this on one line please?

File: Stratis.Bitcoin.Features.Wallet/IWalletManager.cs (hunk 1 above)
Comment (mikedennis) [reply Stratis.Bitcoin.Features.Wallet/IWalletManager.cs]: Yes I like that better. I changed it to ContainsWallets.

**Comment by bokobza** [comment]
Thanks @mikedennis , are you going to do the same for the Consensus line?

**Pull request merged**